sys/arm64/arm64/machdep.c

```diff
@@ -1533,6 +1627,8 @@ do {									\
 	PRINT_REG(vbar_el1);
 #undef PRINT_REG
 #if __has_feature(capabilities)
+	/* TODO: Print other cap registers of interest. */
```

Comment:
Do we think this is still a TODO or should I leave the comment out?

Reply:
Dropping it seems fine.